src/containers/paint-editor-wrapper.jsx

```diff
@@ -40,9 +46,12 @@ class PaintEditorWrapper extends React.Component {
     }
     render () {
         if (!this.props.imageId) return null;
+        const componentProps = omit(this.props, ['selectedCostumeIndex', 'vm']);
```

Comment:
Instead of including lodash.omit,  most of gui uses object destructuring:
```
const {
    selectedCostumeIndex,
    vm,
    ...componentProps
} = this.props;
```
It gets a little messy if you're not using the prop because you need to handle the eslint unused variables rule. In this case you're using both variables, so it should look cleaner.